manifests/rule-evaluator.yaml

```diff
@@ -1,18 +1,11 @@
-# Copyright 2022 Google LLC
-#
```

Comment:
You're losing our license header. Can we add that in through helm?

Reply:
Added `addlicense` tool.